vraygolaem.cpp

```diff
@@ -134,131 +144,87 @@ enum { params, };
 static ParamBlockDesc2 param_blk(params, STR_DLGTITLE,  0, &vrayGolaemClassDesc, P_AUTO_CONSTRUCT+P_AUTO_UI, REFNO_PBLOCK,
 	IDD_VRAYGOLAEM, IDS_VRAYGOLAEM_PARAMS, 0, 0, &vrayGolaemDlgProc,
 	// Params
-	pb_file, _T("cache_file"), TYPE_FILENAME, 0, 0,
+	pb_file, _T("cache_file"), TYPE_FILENAME, P_RESET_DEFAULT, 0,
 		p_ui, TYPE_EDITBOX, ED_GOLAEMVRSCENE,
 #if GET_MAX_RELEASE(VERSION_3DSMAX) >= 11900
 		p_assetTypeID, MaxSDK::AssetManagement::AssetType::kExternalLink,
 #endif
 	PB_END,
-	pb_shaders_file, _T("shaders_file"), TYPE_FILENAME, 0, 0,
+	pb_shaders_file, _T("shaders_file"), TYPE_FILENAME, P_RESET_DEFAULT, 0,
 		p_ui, TYPE_EDITBOX, ED_SHADERSVRSCENE,
 #if GET_MAX_RELEASE(VERSION_3DSMAX) >= 11900
 		p_assetTypeID, MaxSDK::AssetManagement::AssetType::kExternalLink,
 #endif
 	PB_END,
-	pb_use_node_attributes, _T("use_node_attributes"), TYPE_BOOL, 0, 0,
+	pb_use_node_attributes, _T("use_node_attributes"), TYPE_BOOL, P_RESET_DEFAULT, 0,
 	p_default, TRUE,
 	p_ui, TYPE_SINGLECHEKBOX, ED_USERNODEATTRIBUTES,
 	PB_END,
 
 	// display attributes
-	pb_enable_display, _T("enable_display"), TYPE_BOOL, 0, 0,
+	pb_enable_display, _T("enable_display"), TYPE_BOOL, P_RESET_DEFAULT, 0,
 	p_default, TRUE,
 	p_ui, TYPE_SINGLECHEKBOX, ED_DISPLAYENABLE,
 	PB_END,
-	pb_display_percent, _T("display_percent"), TYPE_INT, 0, 0,
+	pb_display_percent, _T("display_percent"), TYPE_INT, P_RESET_DEFAULT, 0,
 	p_default, 100,
 	p_range, 0, 100, 
 	p_ui, TYPE_SPINNER,  EDITTYPE_POS_INT, ED_DISPLAYPERCENT, ED_DISPLAYPERCENTSPIN, 1,
 	PB_END,
-	pb_display_entity_ids, _T("display_entity_ids"), TYPE_BOOL, 0, 0,
+	pb_display_entity_ids, _T("display_entity_ids"), TYPE_BOOL, P_RESET_DEFAULT, 0,
 	p_default, TRUE,
 	p_ui, TYPE_SINGLECHEKBOX, ED_DISPLAYENTITYIDS,
 	PB_END,
 
 	// cache attributes
-	pb_crowd_fields, _T("crowd_fields"), TYPE_STRING, 0, 0,
+	pb_crowd_fields, _T("crowd_fields"), TYPE_STRING, P_RESET_DEFAULT, 0,
 	p_ui, TYPE_EDITBOX, ED_CROWDFIELDS,
 	PB_END,
-	pb_cache_name, _T("cache_name"), TYPE_STRING, 0, 0,
+	pb_cache_name, _T("cache_name"), TYPE_STRING, P_RESET_DEFAULT, 0,
 	p_ui, TYPE_EDITBOX, ED_CACHENAME,
 	PB_END,
-	pb_cache_dir, _T("cache_dir"), TYPE_STRING, 0, 0,
+	pb_cache_dir, _T("cache_dir"), TYPE_STRING, P_RESET_DEFAULT, 0,
 	p_ui, TYPE_EDITBOX, ED_CACHEDIR,
 	PB_END,
-	pb_character_files, _T("character_files"), TYPE_STRING, 0, 0,
+	pb_character_files, _T("character_files"), TYPE_STRING, P_RESET_DEFAULT, 0,
 	p_ui, TYPE_EDITBOX, ED_CHARACTERFILES,
 	PB_END,
-
-	// motion blur attributes
-	pb_motion_blur_enable, _T("motion_blur_enable"), TYPE_BOOL, 0, 0,
```

Comment:
You can't delete parameters from the parameter block; it may cause 3ds Max to crash when loading scenes saved with previous versions of the plugin. Leave the parameters, but set the MaxScript name to be empty (_T("")) and comment out the p_ui tag.